### DISCUSSION TITLE
Player functions

### DISCUSSION THREAD

**Discussion by fax13** [music-assistant/support#905] [Feature requests and ideas]
Hi, what about more media controls? Shuffle,rewind, fore ward next, ect? Would make the player look more like a "music player "

**Comment by OzGav** [comment]
The MA interface already has play, stop, next, and previous and shuffle is enabled on the queue. So not sure what more you are looking for? Is it just fast forward and rewind?

**Reply by fax13** [reply]
It was more an esthetic question so it look more functional when all is centralised  especialy on a cell phone screen
example of grouped control functions
![image](https://user-images.githubusercontent.com/76689575/190923633-b7bee6fc-999b-4896-912f-35dcc51fc614.png)

**Reply by OzGav** [reply]
There is a major rewrite underway but I am yet to see what the front end now looks like. Let’s wait and see what happens there.

**Comment by martinw72** [comment]
It Would be great to be able to group player and synchronise playback from the  player interface!

**Comment by agdamsbo** [comment]
I was thinking about the same here. My designskills are lacking, but here me out :)
I think the interface would be much easier to navigate and interact with on a touch interface, if the menu items within the queue were moved out as icons (text in the picture). Icon color could change depending on whether its active or not (ie shuflle on or off).
![screenshot](https://user-images.githubusercontent.com/43278243/228236798-074a6be3-3a0f-45ae-8309-1fc8765cfd0a.png)

**Reply by OzGav** [reply]
Thanks for the feedback! Yes a major UI redesign is being worked on but the focus for now is on getting the fundamental parts of MA working. From what I have seen you will be very happy with the new UI when it appears in a couple of months.
